src/analysis/Analyses.ts

```diff
@@ -331,7 +331,7 @@ const AnalysisCard = ({
         ...centerColumnFlex,
       },
     },
-    [getFileName(name)]
+    [h(Clickable, { onClick: launchAnalysis }, [getFileName(name)])]
```

Comment:
This was an accessibility issue (a div is not keyboard-focusable), and I believe that the new version of Puppeteer was refusing to click on an item that isn't actually focusable.

Reply:
I have pulled this change into its [own PR](https://github.com/DataBiosphere/terra-ui/pull/5115) because I'm still struggling with some stability issues on this branch.